src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
```

Comment:
>p [](start = 13, length = 1)

You'll need to reserve 'p' for 'pretty-print' on this command. Also, please add 'pretty-print' for the output path. Also, add output file path. Also add a force option to overwrite existing outputs. See SingeFileOptions and the options command base for boilerplate arg definitions and if you see a DRYing opportunity, of course, grab it.

Reply:
Ok. I take it that the output file is the original SARIF file with the bug URIs added.

---
In reply to: [306574608](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306574608) [](ancestors = 306574608)

Reply:
I'd love to use `SingleFileOptions`. But it drags in `CommonOptions` which means I'd have to support `u` (uriBaseId) and `i` (insert optional data) and `v` (select SARIF output version). I'm going to copy the options I need.

---
In reply to: [306577540](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306577540) [](ancestors = 306577540,306574608)

Reply:
oh, it's very unfortunate if you don't support the insert option (and the 'remove' option which is coming. also, we do want uriBaseId.

Can you get your mind around utilizing SingleFileOption. Would be preferable I think. in general, we want all our verbs to provide similar processing capabilities.

---
In reply to: [306578651](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306578651) [](ancestors = 306578651,306577540,306574608)

Reply:
Done. This means I'll have to add code to support the other options. I sure hope there's a single API I can call to parse out the `--insert` option and enrich the SARIF file with the things it asks for.

---
In reply to: [306583029](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306583029) [](ancestors = 306583029,306578651,306577540,306574608)